tfe/data_source_organization_members.go

```diff
@@ -0,0 +1,73 @@
+package tfe
+
+import (
+	"github.com/hashicorp/go-tfe"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceTFEOrganizationMembers() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceTFEOrganizationMembersRead,
+
+		Schema: map[string]*schema.Schema{
+			"organization": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"members": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"user_id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+
+						"organization_membership_id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+					},
+				},
+			},
+
+			"members_waiting": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"user_id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+
+						"organization_membership_id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+					},
+				},
+			},
+		},
+	}
+}
+
+func dataSourceTFEOrganizationMembersRead(d *schema.ResourceData, meta interface{}) error {
+	tfeClient := meta.(*tfe.Client)
+
+	organizationName := d.Get("organization").(string)
+	options := tfe.OrganizationMembershipListOptions{Include: []tfe.OrgMembershipIncludeOpt{tfe.OrgMembershipUser}}
```

Comment:
Since we require only the user ID, there is no need to include users because the normal result only returns the ID. This should make fetching them faster since no extra user data has to be transmitted unnecessarily.

Reply:
Nice, I didn't realize user ID comes by default. 🎉